ipa-core/src/helpers/transport/query/mod.rs

```diff
@@ -234,23 +234,22 @@ impl PartialEq for IpaQueryConfig {
     }
 }
 
-// TODO switch to using an enum
 #[derive(Debug, Copy, Clone)]
-pub enum DPParams {
-    TestingWithNoDP,
-    WithDP(f64),
+pub enum DpParams {
+    NoDp,
+    WithDp(f64),
```

Comment:
```suggestion
    WithDp{
      epsilon: f64,
    }
```

Generally `.epsilon` is easier to read than `.0`.  The effect is the same.

Reply:
okay, yeah that will make it more readable.